platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulersConfig.java

```diff
@@ -84,6 +84,8 @@
  *                                                          garbage collector
  * @param platformPublisher                                 configuration for the platform publisher scheduler
  * @param consensusEventStream                              configuration for the consensus event stream scheduler
+ * @param signedStateSentinel                               configuration for the signed state sentinel scheduler
+ * @param signedStateSentinelHeartbeatPeriod                the frequency that heartbeats should be sent to the signed
```

Comment:
truncated doc

Reply:
tab complete doesn't do more than 1 line at a time in comments.   fixed now.